### PR TITLE
Get Active Blog Post

### PR DESCRIPTION
**Pull request by jcjudkins**
Please make sure to apply the "blog" label to the pull request if it is related to a blog post so that it can be published to Slack. 

## Description

Get Active blog post

### CODE REVIEW

File: _posts/2023-07-07-get-active.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+The [Mountains to Sea Trail](https://mountainstoseatrail.org/the-trail/) runs through Durham if you really want to stretch your legs!
+
+The [Riverwalk](https://www.hillsboroughnc.gov/community/recreation-facilities/riverwalk/) in Hillsborough is a nice walk in the woods.
+
+
+## Bicycling
+
+One rental shop nearby is [Durham Cycles](https://durhamcycles.com/rentals/).
+
+Some routes from a semi-defunct bike shop (lost their lease): [Ride with GPS](https://ridewithgps.com/users/526614/routes).
```
Comment (drewbrew) [inline _posts/2023-07-07-get-active.md:67]:
```suggestion
Some routes from a Durham bike shop which is looking for a new location: [Ride with GPS](https://ridewithgps.com/users/526614/routes).
```

File: _posts/2023-07-07-get-active.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+[Carolina Tarwheels](https://tarwheels.net/wp/) (great name, btw): road riding group with a bunch of routes published from various start points.
+
+The [Triangle MTB](https://www.trianglemtb.com/index.php) site is a good (albeit outdated) hub for MTB stuff in the area.
```
Comment (drewbrew) [inline _posts/2023-07-07-get-active.md:71]:
```suggestion
The [Triangle MTB](https://www.trianglemtb.com/index.php) site is a good (albeit outdated) hub for mountain biking resources in the area. 
```

File: _posts/2023-07-07-get-active.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+Most of the hotels nearby have a fitness center so make sure you check with your hotel if you're staying elsewhere.
+
+## Walking/running/hiking/biking
+
+City of [Durham Bike & Hike Map](https://www.durhamnc.gov/1031/Durham-Bike-Hike-Map) (GIS and PDF forms).
+
+The [North/South Greenway](https://www.dprplaymore.org/265/North-South-Greenway) (7.5 miles of paved trail).
+
+The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) (22 miles of rails to trails).
```
Comment (drewbrew) [inline _posts/2023-07-07-get-active.md:42]:
```suggestion
The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) has 22 miles of rails to trails that starts less than a mile from the hotel and continues south for 14 miles paved and 8 more of gravel after that.
```

**Review by drewbrew — APPROVED** [review]
Looks good, just adding some more spit and polish

File: _posts/2023-07-07-get-active.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+The [Riverwalk](https://www.hillsboroughnc.gov/community/recreation-facilities/riverwalk/) in Hillsborough is a nice walk in the woods.
+
+
+## Bicycling
+
+One rental shop nearby is [Durham Cycles](https://durhamcycles.com/rentals/).
+
+Some routes from a semi-defunct bike shop (lost their lease): [Ride with GPS](https://ridewithgps.com/users/526614/routes).
+
+[Carolina Tarwheels](https://tarwheels.net/wp/) (great name, btw): road riding group with a bunch of routes published from various start points.
```
[the remark this answers is not in the document]
Comment (jcjudkins) [reply _posts/2023-07-07-get-active.md:69]: But it was funny!

**Review by drewbrew — APPROVED** [review]

**Comment by drewbrew** [comment]
Go ahead and merge this today if you want since the schedule likely won't go live until about Thursday

**Comment by drewbrew** [comment]
@jcjudkins we want to introduce the keynotes in the order they're speaking (Abigail, Julia, Andy) if possible. Would it be too hard to get Abigail's up?

**Comment by jcjudkins** [comment]
Pushed up Julia's post for review. 
Started Abigail's post but I am missing some information from her

Moved pending posts to drafts
- Get active
- Pandy Knight
- Julia Solorzano

To do:
- [ ] Need picture and Bio from Abigail
- [ ] Edit Pandy Knight (due to publish order need to change some wording
- [ ] Fix release dates for posts

**Comment by jcjudkins** [comment]
Moved git active post so that it can go live.

File: _drafts/2023-07-07-get-active.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Most of the hotels nearby have a fitness center so make sure you check with your hotel if you're staying elsewhere.
+
+## Walking/running/hiking/biking
+
+City of [Durham Bike & Hike Map](https://www.durhamnc.gov/1031/Durham-Bike-Hike-Map) (GIS and PDF forms).
+
+The [North/South Greenway](https://www.dprplaymore.org/265/North-South-Greenway) (7.5 miles of paved trail).
+
+The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) has 22 miles of rails to trails that starts less than a mile from the hotel and continues south for 14 milespaved and 8 more of gravel after that
```
Comment (drewbrew) [inline _drafts/2023-07-07-get-active.md:42]:
```suggestion
The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) has 22 miles of rails to trails that starts less than a mile from the hotel and continues south for 14 miles of paved trail and 8 more of gravel after that
```

Missed a space

File: _drafts/2023-07-07-get-active.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+The [Falls Lake State Recreation Area](https://www.ncparks.gov/state-parks/falls-lake-state-recreation-area) has 15 miles of mountain biking and 25 miles of hiking. 
+
+<img src="/static/img/blog/philippe-oursel-hEzxeDOMXoo-unsplash-biking.jpg" alt="Photo of bikers by Phillippe Oursel via Unsplash" title="Biking" style="width:30%; display:block; float:right;" />
+
+Eno River park is a 20 minute drive from the hotel and has a lot of hiking trails. [Eno River Association](https://www.enoriver.org/explore/eno-river-state-park/) has a lot of information about the park and trails.
+
+The [Rollingview Marina](https://www.rollingviewmarina.com/kayaks-canoes-paddle-boards/) at Falls Lake has paddle board, kayak, and canoe rentals. 
+
+The town of Cary has a lot of greenways and parks. [Town of Cary Greenways](https://www.townofcary.org/recreation-enjoyment/parks-greenways-environment/greenways) and has boat and Kayak rentals at [Bond Park](https://www.townofcary.org/recreation-enjoyment/facilities/bond-park-boathouse).
```
Comment (drewbrew) [inline _drafts/2023-07-07-get-active.md:54]:
```suggestion
The town of Cary has a lot of [greenways](https://www.townofcary.org/recreation-enjoyment/parks-greenways-environment/greenways) and parks and has boat and kayak rentals at [Bond Park](https://www.townofcary.org/recreation-enjoyment/facilities/bond-park-boathouse).
```

just a minor flow tweak

File: _drafts/2023-07-25-andrew-knight.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+---
+author: Jason Judkins
+category: Program
+date: 2023-07-25 11:00:00-07:00
+image: /static/img/blog/andy_django_profile_zoom_small.jpg
+layout: post
+title: 'Announcing Our Keynotes: Andrew "Pandy" Knight'
+---
+
+### We're honored to announce our first keynote speaker for DjangoCon US 2023: Andrew "Pandy" Knight!
```
Comment (drewbrew) [inline _drafts/2023-07-25-andrew-knight.md:10]:
```suggestion
### We're honored to announce our third keynote speaker for DjangoCon US 2023: Andrew "Pandy" Knight!
```

File: _drafts/2023-08-10-julia-solorzano.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+> - The rise of AI
+> - Cybersecurity
+> - Future Django
+> - Current changes in industry considerations
+> - Conclusion
+
+
+If you are interested in learning more about Julia, you can find her on the web at:
+
+[juliasolorzano.com](https://juliaelman.com/)
```
Comment (drewbrew) [inline _drafts/2023-08-10-julia-solorzano.md:47]:
```suggestion
[juliasolorzano.com](https://juliasolorzano.com/)
```

File: _posts/2023-08-15-get-active.md [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+
+Most of the hotels nearby have a fitness center so make sure you check with your hotel if you're staying elsewhere.
+
+## Walking/running/hiking/biking
+
+City of [Durham Bike & Hike Map](https://www.durhamnc.gov/1031/Durham-Bike-Hike-Map) (GIS and PDF forms).
+
+The [North/South Greenway](https://www.dprplaymore.org/265/North-South-Greenway) (7.5 miles of paved trail).
+
+The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) has 22 miles of rails to trails that starts less than a mile from the hotel and continues south for 14 milespaved and 8 more of gravel after that
```
Comment (drewbrew) [inline _posts/2023-08-15-get-active.md:42]:
```suggestion
The [American Tobacco Trail](https://triangletrails.org/american-tobacco-trail) has 22 miles of rails to trails that starts less than a mile from the hotel and continues south for 14 miles of paved trail and 8 more of gravel after that.
```

File: _posts/2023-08-15-get-active.md [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+
+The [Falls Lake State Recreation Area](https://www.ncparks.gov/state-parks/falls-lake-state-recreation-area) has 15 miles of mountain biking and 25 miles of hiking. 
+
+<img src="/static/img/blog/philippe-oursel-hEzxeDOMXoo-unsplash-biking.jpg" alt="Photo of bikers by Phillippe Oursel via Unsplash" title="Biking" style="width:30%; display:block; float:right;" />
+
+Eno River park is a 20 minute drive from the hotel and has a lot of hiking trails. [Eno River Association](https://www.enoriver.org/explore/eno-river-state-park/) has a lot of information about the park and trails.
+
+The [Rollingview Marina](https://www.rollingviewmarina.com/kayaks-canoes-paddle-boards/) at Falls Lake has paddle board, kayak, and canoe rentals. 
+
+The town of Cary has a lot of greenways and parks. [Town of Cary Greenways](https://www.townofcary.org/recreation-enjoyment/parks-greenways-environment/greenways) and has boat and Kayak rentals at [Bond Park](https://www.townofcary.org/recreation-enjoyment/facilities/bond-park-boathouse).
```
Comment (drewbrew) [inline _posts/2023-08-15-get-active.md:54]:
```suggestion
The town of Cary has a lot of [greenways](https://www.townofcary.org/recreation-enjoyment/parks-greenways-environment/greenways) and parks and has boat and kayak rentals at [Bond Park](https://www.townofcary.org/recreation-enjoyment/facilities/bond-park-boathouse).
```

**Review by drewbrew — APPROVED** [review]
looks good with a couple of minor tweaks

**Comment by jcjudkins** [comment]
Updated suggestions!

**Pull request merged**